liquibase-standard/src/main/java/liquibase/util/StringUtil.java

```diff
@@ -33,6 +34,27 @@ public static String trimToEmpty(String string) {
         return string.trim();
     }
 
+    /**
+     *
+     * Split a string, trim each part, and return a new array
+     *
+     * @param   string           String to operate on
+     * @return  String[]
+     *
+     */
+    public static String[] splitAndTrim(String string) {
+        String[] newParts = string.split(",");
+        if (newParts.length == 1) {
+            return newParts;
+        }
+        String[] editedParts = new String[newParts.length];
+        AtomicInteger i = new AtomicInteger();
+        Arrays.stream(newParts).forEach(label-> {
+            editedParts[i.getAndIncrement()] = label.trim();
+        });
+        return editedParts;
+    }
```

Comment:
This method already exists on line 421. Please use that one instead and remove this.